tests/test_wikipron/__init__.py

```diff
@@ -1,12 +1,14 @@
+from typing import Dict
+
 import requests
 
-from wikipron.scrape import HTTP_HEADERS
 from wikipron.config import Config
+from wikipron.scrape import HTTP_HEADERS
 
 
 def config_factory(**kwargs) -> Config:
     """Create a Config object for testing."""
-    config_dict = {"key": "eng"}  # The one default; may be overridden.
+    config_dict: Dict = {"key": "eng"}  # The one default; may be overridden.
```

Comment:
I didn't know this worked, don't you have to say `Dict[str, str]` or is that the default?

Reply:
I think this was enough to satisfy mypy, but I agree it's not explicit enough.